virtool/startup.py

```diff
@@ -198,7 +201,10 @@ async def startup_databases(app: App):
 
 async def startup_routes(app: App):
     logger.debug("Setting up routes")
-    setup_routes(app, dev=app["config"].dev)
+
+    config = app["config"]
+
+    setup_routes(app, dev=config.dev if hasattr(config, "dev") else False)
```

Comment:
Why the change? dev attribute should always be set if the startup path is getting called

Reply:
Some processes in tilt complained `app["config"] as no attribute "dev"`